src/legacy/core_plugins/kibana/public/discover/np_ready/components/sidebar/discover_field_bucket.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { EuiButtonIcon, EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { StringFieldProgressBar } from './string_progress_bar';
+import { getBucketAriaLabel } from './lib/get_bucket_aria_label';
+import { Bucket } from './types';
+import { IndexPatternField } from '../../../../../../../../plugins/data/public';
+
+interface Props {
+  bucket: Bucket;
+  field: IndexPatternField;
+  onAddFilter: (field: IndexPatternField | string, value: string, type: '+' | '-') => void;
+}
+
+export function DiscoverFieldBucket({ field, bucket, onAddFilter }: Props) {
+  const emptyTxt = i18n.translate('kbn.discover.fieldChooser.detailViews.emptyStringText', {
+    defaultMessage: 'Empty string',
+  });
+  const addLabel = i18n.translate(
+    'kbn.discover.fieldChooser.detailViews.filterValueButtonAriaLabel',
+    {
+      defaultMessage: 'Filter for this value',
+    }
+  );
+  const removeLabel = i18n.translate(
+    'kbn.discover.fieldChooser.detailViews.filterOutValueButtonAriaLabel',
+    {
+      defaultMessage: 'Filter out this value',
```

Comment:
I changed it to `Filter out {fieldname}: "{value}"`
dear @gchaps , does that sound ok?

Reply:
@kertal Can you please give me an example of how this message will read with values for {fieldname} and {value}?

I'm leaning more toward something that might be a littler easier to read "Filter out value for {fieldname}.

Reply:
currently it would be 
```
Exlusive Filter: Filter out extension: "php"
Inclusive Filter: Filter for extension: "php"
```
that would change to 
```
Exlusive Filter: Filter out "php" for extension
Inclusive Filter: Filter for "php" for(?) extension
```

Reply:
I like the one on top:

Filter out extension: "php"
Filter for extension: "php"

Reply:
already none :)